esper/ext/remoteadb_api.py

```diff
@@ -104,6 +105,15 @@ def fetch_relay_endpoint(environment: str,
         if is_ok:
             host = remoteadb_session.get("ip")
             port = remoteadb_session.get("client_port")
+            remoteadb_host = remoteadb_session.get("remoteadb_host")
+
+            if remoteadb_host:
+                try:
+                    ip_address = socket.gethostbyname(remoteadb_host)
+                    host = ip_address
+                except socket.error as e:
+                    log.debug(f"[remoteadb-connect] Could not resolve ip address of '{remoteadb_host}', falling back to {host}. Error was: {e}")
```

Comment:
We are not falling back to `host` implementation right? Could you edit the log accordingly?

Reply:
Resolved. Also, used `error` instead of `debug`.